tests/Aspire.Hosting.RabbitMQ.Tests/AddRabbitMQTests.cs

```diff
@@ -163,7 +163,6 @@ public void WithManagementPluginThrowsForUnsupportedContainerImageTag(string ima
     }
 
     [Theory]
-    [InlineData(" ")]
```

Comment:
Why this change?

Reply:
`rabbitmq.WithImage(" ")` now throws `invalid reference format` before the test can even get as far as testing if `rabbitmq.WithManagementPlugin` throws.

This scenario is covered by https://github.com/dotnet/aspire/blob/03b07232818f172246dbc274b3fc1fa6a9a07cb6/tests/Aspire.Hosting.Tests/Utils/ContainerReferenceParserTests.cs#L20

Reply:
@mitchdenny does my explanation resolve your question about the test change?